conan/tools/pkg_signing/plugin.py

```diff
@@ -0,0 +1,64 @@
+import copy
+import os
+import json
+
+from conan.internal.util.files import load, sha256sum, save
+
+# FIXME: Maybe this tools should be placed at conan.api.subapi.cache as they are not recipe tools?
+
+SIGN_SUMMARY_CONTENT = {
+    "provider": None,
+    "method": None,
+    "files": {}
+}
+
+SIGN_SUMMARY_FILENAME = "sign-summary.json"
+
+
+def get_summary_file_path(signature_folder):
```

Comment:
So finally these tools are "recommendations" or available for users, but not enforced by Conan at all, users could be doing any signing the way they like, without summary paths, etc?